.github/workflows/ci.yml

```diff
@@ -9,10 +9,13 @@ on:
   workflow_dispatch:
 
 jobs:
-  # This workflow contains a single job called "build-v1"
   build-v1:
+    strategy:
+      matrix:
+        # Run this workflow on different os
+        os: [windows-latest, ubuntu-22.04, macos-latest]
```

Comment:
We pin to ubuntu-22.04. Should we also pin to particular versions of Windows and macOS?

Reply:
Sure, do we have a recommended version? if not, what about windows-2022 and macos-12?